README.md

```diff
@@ -61,38 +74,73 @@ Where you see params it is a plain js object, e.g. `{page: 1}`. For a list of pa
 * page
   * `retrieve(page_type, page_slug[, params])`
   * `list(page_type[, params])`
+  * `search(query[, params])`
+  * `cancelRequest()`
   
  ```js
 // Get page
-butter.page.retrieve('casestudy', 'acme-co').then(function(resp) {
+butter.page.retrieve("casestudy", "acme-co").then(function(resp) {
   console.log(resp)
 });
 ```
 
+If using `async`/`await`:
+
+```js
+async function getPage() {
+  try {
+    const response = await butter.page.retrieve("casestudy", "acme-co");
+    console.log(response);
+  } catch (error) {
+    console.error(error);
+  }
+}
+```
+
 ## Collections
 
 * content
   * `retrieve(collection[, params])`
-  
+  * `cancelRequest()`
+
 ```js
 // Get FAQ
-butter.content.retrieve(["faq"], {locale: 'es'}).then(function(resp) {
+butter.content.retrieve(["faq"], {locale: "es"}).then(function(resp) {
   console.log(resp)
 });
 ```
+If using `async`/`await`:
+
+```js
+async function getFAQ() {
+  try {
+    const response = await butter.content.retrieve(["faq"], {locale: "es"});
```

Comment:
I thought we wanted to remove the usage of multiple slugs/keys and allow just one